src/matrix.hpp

```diff
@@ -195,6 +201,31 @@ struct matrix {
             [](const V& m0_col, const V& m1_col) { return m0_col - m1_col; });
         return m2;
     }
+
+    [[nodiscard]] constexpr auto transpose() const
+        -> matrix<typename extend_to<V, cols>::type, rows>
+    {
+
+        return [this]() {
+            matrix<typename extend_to<V, cols>::type, rows> t{};
+            // Cycle over rows
+            [ this, &t ]<row_index... Rs>(row_seq<Rs...>)
+            {
+                (
+                    // Cycle over columns
+                    [ this, &t ]<std::size_t Ras, col_index... Cs>(
```

Comment:
What happened to the stuff we chatted about over the weekend? Is that in a different commit?

Reply:
It's a bit down the chain, in "matrix entries". But I see I might have had an old version. Let's review that as soon as we get there